app/templates/editor/level/tasks-tab.jade

```diff
@@ -0,0 +1,41 @@
+mixin task-row(task)
+  - var taskName = task.get('name');
+  - var isComplete = task.get('complete')
+  - var taskLink = view.defaultTaskLinks[taskName]
+  tr.taskRow(data-task=task)
+    td.taskCheck
+      div.checkbox
+        input(type='checkbox', checked=(isComplete || false)).taskInput
+    if task.get('curEdit') === true
+      td.editCell
+      td
+        input(type="input", value=taskName)#curEdit
+    else
+      td.editCell
+        span.startEdit ✎
```

Comment:
Use glyphicons rather than unicode. If for no other reason than to be consistent.

http://getbootstrap.com/components/#glyphicons
`span.glyphicon.glyphicon-pencil`